File: Fruits Into Baskets II/Fruits_Into_Baskets_II.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        n = len(fruits)
        unplace = 0
        for i in range(0,n):
            flag = False
            for j in range(0,n):
                if baskets[j] >= fruits[i]:
                    baskets[j] = 0;
                    flag = True
                    break
                
            
            if flag == False:
                unplace +=1
        
        return unplace;
        

File: Fruits Into Baskets II/Fruits_Into_Baskets_II.cpp
class Solution {
public:
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int n = fruits.size();
        int unplace = 0;
        for(int i =0; i<n; i++) {
            bool flag = false;
            for(int j = 0; j<n; j++) {
                if(baskets[j] >= fruits[i]) {
                    baskets[j] = 0;
                    flag = true;
                    break;
                }
            }
            if(flag == false) unplace++;
        }
        return unplace;
    }
};